src/Adapters/Laravel/Repositories/DatabaseAnalyticsRepository.php

```diff
@@ -58,4 +58,14 @@ public function flush(): void
     {
         DB::table('pan_analytics')->truncate();
     }
+
+    /**
+     * Delete a specific analytic by ID.
+     */
+    public function delete(int $id): string
+    {
+        return DB::table('pan_analytics')->where('id', $id)->delete()
```

Comment:
I think the client i.e `the command` should be the one handling this. Returning void/int here would be better than this string

```suggestion
        return DB::table('pan_analytics')->where('id', $id)->delete()
```